documentation/API proposals/APIproposal_SessionInsights_CableLabs.md

```diff
@@ -1,16 +1,16 @@
+| Technical viability                       | Application developers can use this API to interact with the network. It assumes the network operator exposing this API has a backend to receive KPIs, analyze them, and produce a Session Insights score. Based on minimum quality thresholds for the application category, the network operator can provide proactive recommendations. The application could also use the CAMARA NetworkAccessManagement for remote control of a [`Device`](https://github.com/camaraproject/NetworkAccessManagement/blob/1c804a5159f67370f005dbf0cc82e5c7e72725f0/code/API_definitions/network_access_management.yaml#L34) at a [`Service Site`](https://github.com/camaraproject/NetworkAccessManagement/blob/1c804a5159f67370f005dbf0cc82e5c7e72725f0/code/API_definitions/network_access_management.yaml#L40) if necessary.                                                               |
+| Commercial viability                      | Reference implementation developed by CableLabs with a sample video conferencing app using the API is being evaluated with network operators.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| YAML code available?                      | YES<br>- [yaml OpenAPI Specification](../SupportingDocuments/sessioninsights-openapi.yaml)<br>- [yaml - Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/cablelabs/APIBacklog/benhepworth-patch-2/documentation/SupportingDocuments/sessioninsights-openapi.yaml)<br>- [slides](../SupportingDocuments/session-insights-api.pdf)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
The link to the PDF might better fit into another field, just a link to further information?

Reply:
makes sense - moving it into the API summary section.